test/e2e/suite/conformance/certificates/venafi/venafi.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/jetstack/cert-manager/test/e2e/suite/conformance/certificates"
 )
 
-var _ = framework.ConformanceDescribe("[Feature:Issuers:Venafi:TPP] Certificates", func() {
```

Comment:
Hmm this is surprising to me. Should be not be controlling what tests are run via test runner configs, rather than changing the tests themselves?

I would have thought we would add a skip string to ginkgo to control whether the TPP tests are run or not.

Reply:
I think in future we absolutely should do this in jetstack/testing - I agree that this is the incorrect way of doing things

All this PR does is revert this original commit https://github.com/jetstack/cert-manager/commit/cc0b6d44556b115f6b55f361b4def32065684e14

Reply:
I just saw that this is "requested changes" and not a "comment"

To be clear this is _reverting_ the thing you think is mistaken. I'm not sure what you want me to change. I think the PR description is pretty clear on this.

Reply:
The mechanism was copied from Kubernetes. See
* https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/flaky-tests.md#quarantining-flakes
* https://github.com/jetstack/cert-manager/pull/3679/files

I personally find it easier to disable E2E tests here in the cert-manager repository, than trying to remember how to do it in the testing repo (see https://github.com/jetstack/testing/pull/409) and because PRs in the testing repo don't allow me to check, before merging, whether the changes I have made will have the desired effect....at least I don't know of a way.

Reply:
And here for example, where we're trying to figure out whether the TPP server is working again. We can check that in a branch / PR without disruption. If we re-enabled the tests globally in the testing repo, everyone's PRs would be unmergable until we figured out the TPP problem.